SU2_CFD/src/iteration_structure.cpp

```diff
@@ -494,95 +494,46 @@ void CMeanFlowIteration::Iterate(COutput *output,
                                  CFreeFormDefBox*** FFDBox,
```

Comment:
Yep, the inner loop is now at driver level. 
In the previous implementation internal iterations were carried out separately for each zone. This meant that the code was enrolling the internal loop first for zone 1, then for zone 2 and so on. In this way variables transfer at interfaces basically occurred only after advancing the physical time and the problem was loosely coupled.
In the new implementation, transfer classes are updated at each pseudo time step creating a stronger coupling and improving convergence.